src/utils/api.js

```diff
@@ -65,3 +65,16 @@ export const getCommentsByArticleId = (articleId) => {
       return err;
     });
 };
+
+export const updateArticleVotes = (articleId, voteValue) => {
+  return axios
+    .patch(`https://thenews-lhhv.onrender.com/api/articles/${articleId}`, {
```

Comment:
have a look at axios.create, it could be a nice refactor for this utils file

Reply:
thanks a million i ll dive into it ! 

https://blog.logrocket.com/understanding-axios-create/